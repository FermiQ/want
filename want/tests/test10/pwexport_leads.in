&inputpp
  prefix='chain4'
  outdir='./LEADS/'
  ascii = .FALSE.
  pseudo_dir = './'
  psfile(1) = 'C.blyp-mt.UPF'
  single_file = .FALSE.
  ascii = .FALSE.
/
