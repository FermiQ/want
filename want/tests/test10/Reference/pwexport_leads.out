
     Program POST-PROC v.3.2cvs starts ...
     Today is 14Oct2006 at  7:53:24 

     nbndx  =    40  nbnd   =    40  natomwfc =    16  npwx   =    8539
     nelec  =  16.00  nkb   =    32  ngl    =    7151

     negative rho (up, down):  0.443E-02 0.000E+00
